src/app.ts

```diff
@@ -53,6 +54,16 @@ program
       "smaller than the given number of lines." +
       0,
   )
+  .option(
+    "-o, --output-format <format>",
+    "Specifies what format the output should be, current options are: terminal/console, json, html.",
+    "terminal",
+  )
+  .option(
+    "-v, --cluster-cut-off-value <number>",
+    "The minimum amount of lines needed before two files will be clustered together",
+    13,
```

Comment:
wouldn't it be better to use a percentage score (i.e. the score value) here so it doesn't depend on file size?

Reply:
I though about that but I think doing it my way would be slightly more clear and allows the users to more easily pick a value that is relevant for them.